### PR TITLE
remove cluster permissions for secrets

### PR DESCRIPTION
**Pull request by ppyt-pl**

### CODE REVIEW

**Comment by ppyt-pl** [comment]
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/48838

**Comment by ppyt-pl** [comment]
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/48838

File: bundle/manifests/ibm-crossplane-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -966,7 +959,6 @@ spec:
   minKubeVersion: 1.19.0
   provider:
     name: IBM
-  replaces: ibm-crossplane-operator.v1.0.0
```
Comment (arturobrzut) [inline bundle/manifests/ibm-crossplane-operator.clusterserviceversion.yaml:969]: replace should be

**Review by arturobrzut — CHANGES_REQUESTED** [review]

File: bundle/manifests/ibm-crossplane-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (ppyt-pl) [reply bundle/manifests/ibm-crossplane-operator.clusterserviceversion.yaml]: done

**Review by arturobrzut — APPROVED** [review]

**Pull request merged**